packages/svelte/src/internal/client/dom/elements/attributes.js

```diff
@@ -347,15 +347,27 @@ export function set_attributes(
 		} else if (key === '__value' || (key === 'value' && value != null)) {
 			// @ts-ignore
 			element.value = element[key] = element.__value = value;
+		} else if (key === 'defaultValue') {
+			/** @type {HTMLInputElement} */ (element).defaultValue = value;
+		} else if (key === 'defaultChecked') {
+			/** @type {HTMLInputElement} */ (element).defaultChecked = value;
```

Comment:
I think we can remove those in favor of doing `value == null && !is_custom_element && key !== 'defaultValue' && key === 'defaultChecked'` below

Reply:
I think we can't because we also want to nullify it if the value is nullish right?

Reply:
Oh no wait it's because in the `else if` we are checking for `setters.include`